doc/classes/AStarGrid2D.xml

```diff
@@ -12,105 +23,156 @@
 			</description>
 		</method>
 		<method name="_estimate_cost" qualifiers="virtual const">
 			<return type="float" />
 			<param index="0" name="from_id" type="Vector2i" />
 			<param index="1" name="to_id" type="Vector2i" />
 			<description>
+				Called when estimating the cost between a point and the path's ending point.
+				Note that this function is hidden in the default [code]AStarGrid2D[/code] class.
 			</description>
 		</method>
 		<method name="clear">
 			<return type="void" />
 			<description>
+				Clears the grid and sets the [member size] to [constant Vector2i.ZERO].
 			</description>
 		</method>
 		<method name="get_id_path">
 			<return type="PackedVector2Array" />
 			<param index="0" name="from_id" type="Vector2i" />
 			<param index="1" name="to_id" type="Vector2i" />
 			<description>
+				Returns an array with the IDs of the points that form the path found by AStar2D between the given points. The array is ordered from the starting point to the ending point of the path.
 			</description>
 		</method>
 		<method name="get_point_path">
 			<return type="PackedVector2Array" />
 			<param index="0" name="from_id" type="Vector2i" />
 			<param index="1" name="to_id" type="Vector2i" />
 			<description>
+				Returns an array with the points that are in the path found by AStarGrid2D between the given points. The array is ordered from the starting point to the ending point of the path.
+				[b]Note:[/b] This method is not thread-safe. If called from a [Thread], it will return an empty [PackedVector3Array] and will print an error message.
 			</description>
 		</method>
 		<method name="is_dirty" qualifiers="const">
 			<return type="bool" />
 			<description>
+				Indicates that the grid parameters was changed and the [method update] needs to be called.
 			</description>
 		</method>
 		<method name="is_in_bounds" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="x" type="int" />
 			<param index="1" name="y" type="int" />
 			<description>
+				Returns [code]true[/code] if the [param x] and [param y] is a valid grid coordinate (id).
 			</description>
 		</method>
 		<method name="is_in_boundsv" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="id" type="Vector2i" />
 			<description>
+				Returns [code]true[/code] if the [param id] vector is a valid grid coordinate.
 			</description>
 		</method>
 		<method name="is_point_solid" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="id" type="Vector2i" />
 			<description>
+				Returns whether a point is disabled or not for pathfinding. By default, all points are enabled.
 			</description>
 		</method>
 		<method name="set_point_solid">
 			<return type="void" />
 			<param index="0" name="id" type="Vector2i" />
 			<param index="1" name="solid" type="bool" default="true" />
 			<description>
+				Disables or enables the specified point for pathfinding. Useful for making a temporary obstacle.
```

Comment:
```suggestion
				Disables or enables the specified point for pathfinding. Useful for making a temporary obstacle. By default, all points are enabled.
```
Why only for temporary obstacles? What would be the best way to make permanent obstacles?

Reply:
Fine, removed `temporary` term